Tools/Pipeline/Common/PipelineTypes.cs

```diff
@@ -408,6 +408,9 @@ private static void ResolveAssemblies(IEnumerable<string> assemblyPaths)
                 try
 #endif
                 {
+                    if (!asm.ToString().Contains("MonoGame"))
+                        continue;
```

Comment:
This would fail on any custom pipeline extensions that don't include "MonoGame" in the file name.

We need a better way to filter this.